MainModule/Client/Plugins/Anti_Cheat.lua

```diff
@@ -151,6 +151,15 @@ return function(Vargs)
 	end
 
 	local Detectors = {
+		Anti32bit = function()
+			if service.UserInputService.TouchEnabled and not service.UserInputService.KeyboardEnabled and not service.UserInputService.MouseEnabled then return end -- Do not detect on mobile
```

Comment:
This does not stop the Xbox version from being falsely detected.
You should add `and not GuiService:IsTenFootInterface()` to prevent Xbox from being detected.

Reply:
Taking a guess that Xboxes run 32-bit clients due to the fact that the 64-bit clients are rather new. Will implement soon.

Reply:
Superseded by [commit #7f8398a](https://github.com/Epix-Incorporated/Adonis/pull/1186/commits/7f8398a82ef5d903719d9729b158f495d9e854a6)